WooCommerce/Classes/ViewRelated/Coupons/CouponDetails/CouponDetails.swift

```diff
@@ -172,11 +172,7 @@ struct CouponDetails: View {
                                         Text(amount)
                                             .font(.title)
                                     } else {
-                                        // Shimmering effect on mock data
-                                        Text("$0.00")
-                                            .font(.title)
-                                            .redacted(reason: .placeholder)
-                                            .shimmering()
+                                        ActivityIndicator(isAnimating: .constant(true), style: .medium)
```

Comment:
This is necessary because the Shimmer library is causing a bug similar to [this](https://github.com/woocommerce/woocommerce-ios/issues/11611).